backend/src/providers/transactions.ts

```diff
@@ -158,6 +283,12 @@ export const getTransactionDetails = async (
     fee: transactionFee.toString(),
     amount: transactionAmount.toString(),
     status: convertRpcTxStatusToStatus(transactionInfo.status),
+    gas: {
+      used: gasAttached.toString(),
+      attached: gasUsed.toString(),
+    },
```

Comment:
Actually we don’t use it, and we can delete it

Reply:
Let's do it? :)